package.json

```diff
@@ -50,7 +50,7 @@
     "winston": "3.9.0"
   },
   "devDependencies": {
-    "@nypl/design-system-react-components": "3.1.2",
+    "@nypl/design-system-react-components": "3.1.7",
```

Comment:
Can you include this in the changelog as well. The Pr is already merged but in the next PR or in the release branch.